packages/rpc/src/SocketClient.ts

```diff
@@ -25,7 +25,11 @@ class SocketClient extends RpcClientBase {
   private _client: WebsocketClient | null = null;
 
   public connect(): void {
-    this._client = new WebsocketClient(this.endpoint, (e: Error) => {
```

Comment:
NOTE: This file is going away completely, so I am just parsing the RPC HTTP URL here to force it into the appropriate websocket URL, but we won't be using websockets soon, this is just temporary.